config/opensearch_dashboards.yml

```diff
@@ -76,6 +76,13 @@
 # must be a positive integer.
 #opensearch.requestTimeout: 30000
 
+
+# Set the value of this setting to true to enable memory circuit breaker for large query response
+#opensearch.memoryCircuitBreaker.enabled: false
+
+# The pecentage of maximum heap allowed for processing response. This value must be within [0.6, 1.0]. For example, '0.8'.
```

Comment:
I think we should add more helpful information here so that it's clear how this might impact a node/cluster.

Reply:
This should improve the stability of OSD in general if enabled, the value is tunable based on needs. I feel i should remove second half of the comment because that is the recommendation for Juno (60% is the threshold for container level auto scaling).